test/utils/deriveAddress/index.ts

```diff
@@ -0,0 +1,11 @@
+import assert from 'assert-diff'
+import {deriveAddress} from 'ripple-keypairs/dist'
+
+describe('Derive Address', function () {
```

Comment:
We don't need to test ripple-keypairs code here, it's already tested there - this can be deleted